install/kubernetes/helm/istio-remote/charts/sidecarInjectorWebhook/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: istio-sidecar-injector-admin-role-binding-{{ .Release.Namespace }}
+  labels:
+    app: istio-sidecar-injector
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: istio-sidecar-injector-{{ .Release.Namespace }}
+subjects:
+  - kind: ServiceAccount
+{{- if .Values.global.rbacEnabled }}
```

Comment:
I was a bit confused, seems here the rbac logic in your PR only handling the serviceAccount name? 

Same issue for the local chart. @ymesika as you initiated the original PR for local chart, any comments for this?

Reply:
I don't think its helpful to fix bugs in the main chart by reviewing istio-remote :)  It may make more sense to just fix those problems independently and use a copy and paste as was done here (and was done in the past for citadel).

Reply:
make sense.